### PR TITLE
Add DicussionCommentEvent Webhook

### PR DESCRIPTION
**Pull request by apurwaj2**
This commit has changes to enable support to receive DiscussionComment Webhook Event, to fix [google#2497](url) Changes include:
* Addition of event type for this type of event DiscussionCommentEvent
* Changes for identification and mapping of DiscussionCommentEvent
* Addition / Modification of relevant tests
* Addition of files created by go generate

### CODE REVIEW

**Comment by gmlewis** [comment]
Please run your files through "gofmt" as described in CONTRIBUTING.md.

File: github/event_types.go [hunk 1]
```diff
@@ -189,6 +189,39 @@ type DeploymentStatusEvent struct {
...
+	AuthorAssociation  *string             `json:"author_association,omitempty"`
+	Body               *string             `json:"body,omitempty"`
+	ChildCommentCount  *int				   `json:"child_comment_count,omitempty"`
+	CreatedAt          *Timestamp          `json:"created_at,omitempty"`
+	DiscussionID       *int64			   `json:"discussion_id,omitempty"`
+	HTMLURL            *string             `json:"html_url,omitempty"`
+	ID                 *int64              `json:"id,omitempty"`
+	NodeID             *string             `json:"node_id,omitempty"`
+	ParentID		   *int64			   `json:"parent_id,omitempty"`
+	Reaction           *Reactions          `json:"reactions,omitempty"`
```
Comment (gmlewis) [inline github/event_types.go:219]:
```suggestion
	Reactions           *Reactions          `json:"reactions,omitempty"`
```

Comment (apurwaj2) [reply github/event_types.go]: Addressed in the new commit. Thank you.

**Review by gmlewis — APPROVED** [review]
Thank you, @apurwaj2 !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

Note that I'm OOO until around March 6, so I may not be able to merge this before then if we don't get an approval shortly.

**Comment by apurwaj2** [comment]
> Thank you, @apurwaj2 ! LGTM.
> 
> Awaiting second LGTM+Approval from any other contributor to this repo before merging.
> 
> Note that I'm OOO until around March 6, so I may not be able to merge this before then if we don't get an approval shortly.

Sure. Thanks, for approving.

**Review by valbeat — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**